robosuite/utils/control_utils.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import robosuite.utils.transform_utils as trans
+import scipy.linalg
+
+
+def nullspace_torques(mass_matrix, nullspace_matrix, initial_joint, joint_pos, joint_vel):
```

Comment:
please add docstring and comments on what this function does (talk about the nullspace controller that tries to maintain a secondary objective of maintaining a desired joint position)

Reply:
@cremebrule I wanted to follow up on this - do you need any help here?

Reply:
sorry -- missed all of these comments that were compressed when scrolling through them! done. Please feel free to double-check the docstrings I added and add anything you feel is missing / unclear